app/src/main/java/de/westnordost/streetcomplete/screens/settings/SettingsViewModel.kt

```diff
@@ -31,7 +28,7 @@ import kotlinx.coroutines.flow.StateFlow
 abstract class SettingsViewModel : ViewModel() {
     abstract val selectableLanguageCodes: StateFlow<List<String>?>
     abstract val selectedQuestPresetName: StateFlow<String?>
-    abstract val hiddenQuestCount: StateFlow<Long>
+    abstract val hiddenQuestCount: StateFlow<Int>
```

Comment:
What if I want to hide more than 2,147,483,647 quests now? :open_mouth: :wink:

Reply:
Don't worry, if you hide 10 quests per second we still have a few years to fix it